transfuse-core/src/main/java/org/androidtransfuse/util/apache/commons/WriterOutputStream.java

```diff
@@ -0,0 +1,88 @@
+package org.androidtransfuse.util.apache.commons;
```

Comment:
I'd stick this in `org.androidtransfuse.gen` next to `FilerSourceCodeWriter`

Reply:
done, just moved next to FilerSourceCodeWriter